tests/network/network_simplification/test_network_graph_simplificator.py

```diff
@@ -132,26 +139,30 @@ def _get_network_simplification_with_attribute_exclusion(
     def _get_nx_digraph_factory(self) -> Iterator[Callable[[], nx.MultiDiGraph]]:
         def create_nx_multidigraph():
             _nx_digraph = nx.MultiDiGraph()
-            for i in range(1, 16):
+            for i in range(1, 19):
```

Comment:
You can easily add multiple nodes from a list with `add_nodes_from`

Reply:
modified